packages/core/scripts/local/DeployEMP.js

```diff
@@ -26,9 +26,13 @@ const { interfaceName, ZERO_ADDRESS, parseFixed } = require("@uma/common");
 const { getAbi, getTruffleContract } = require("../../index");
 const argv = require("minimist")(process.argv.slice(), {
   boolean: ["test"],
-  string: ["identifier", "collateral", "cversion"]
+  string: ["identifier", "collateral", "cversion", "name", "symbol", "expiry"]
 });
 const abiVersion = argv.cversion || "1.2.2"; // Default to most recent mainnet deployment, 1.2.2.
+const syntheticName = argv.name || "Test Synth";
+const syntheticSymbol = argv.symbol || "SYNTH";
+const expiry = argv.expiry || 2;
+const expirationTimestamp = Math.ceil(Date.now() / 1000) + Number(expiry); // 2 minutes from now
```

Comment:
```suggestion
const expirationTimestamp = Math.ceil(Date.now() / 1000) + Number(expiry);
```

Reply:
i think the naming of expiry is confusing, i think of it a date in the future, but its being used as a duration. Maybe might be better called duration or secondsTillExpiry or something that names the units so its a little more clear.

Reply:
also if expiry is meant to be minutes it should be multiplied by 60

Reply:
Yeah, this makes sense to me.